src/plays/url-shortner/UrlShortner.jsx

```diff
@@ -73,25 +83,32 @@ function UrlShortner(props) {
                   }}
                 />
                 <button
-                  class="ms-4 group relative h-12 w-48 overflow-hidden rounded-lg bg-white text-lg shadow"
+                  class="ms-4 group mt-4 relative h-12 px-6 overflow-hidden rounded-lg bg-white text-lg shadow"
                   onClick={handleSubmit}
                 >
                   <div class="absolute inset-0 w-3 bg-[#0087fe] transition-all duration-[250ms] ease-out group-hover:w-full" />
-                  <span class="relative text-black group-hover:text-white">Shorten This Link</span>
+                  <span class="relative text-black group-hover:text-white">Short URL</span>
                 </button>
               </div>
-              <div className="mt-5">
-                {error && <div className="text-red-500 mb-2">{error}</div>}
-                <div className="relative px-4 py-2 bg-white shadow-lg rounded-xl  bg-clip-padding bg-opacity-60 border border-gray-200 text-slate-500">
-                  {shortenedLink ? shortenedLink : 'Shorten An URL First'}
+              {error && <div className="text-red-500 mt-5">{error}</div>}
+              {shortenedLink && (
+                <div className="mt-5">
+                  <p>Here's your shortened URL:</p>
+
+                  <div className="flex justify-center items-center">
+                    <div className="relative px-4 py-2 bg-white shadow-lg rounded-xl  bg-clip-padding bg-opacity-60 border border-gray-200 text-slate-500">
+                      {shortenedLink ? shortenedLink : 'Shorten An URL First'}
+                    </div>
+                    <button
+                      className="border-2 px-5 py-2 ml-4 rounded-md font-medium 'border-blue-500 text-blue-500 hover:bg-blue-500 hover:text-white cursor-pointer"
+                      disabled={!shortenedLink}
```

Comment:
We are already checking at the top that `shortenedLink` is available, so we don't need this piece of code. `disabled={!shortenedLink}`